Jenkinsfile

```diff
@@ -32,6 +32,12 @@ pipeline {
                     env.DISTRIBUTION_BUILD = env.BRANCH_NAME.startsWith('PR-') && !pullRequest.labels.contains('Documentation') ? "" : "-Pdistribution"
                 }
 
+                // Collect reports on non-prs
+                script {
+                    //env.REPORTS_BUILD = env.BRANCH_NAME.startsWith('PR-') ? "" : "site"
+                    env.REPORTS_BUILD = "site"
```

Comment:
Could we get away without building the site? The HTML output isn't usable as is anyway, we'd need to run the `site:deploy` or `site:stage` goals to put them in the correct directory structure (the links from the parent use the artifact id as the subdirectory name instead of the source directory name).

Reply:
I'm now just building the individual reports.